### PR TITLE
Fix/Mini-1694, 1771

### PR DESCRIPTION
**Pull request by rleojoseph**
# Description
* Fix - Remove Trailing whitespace when the user taps on Save button
* Fix - No need to validate User name in User settings - _"No field is explicitly required."_ 
* Delete existing Custom permissions for the miniapp from the Keychain when it is downloaded for the first time
* Refactor - Show Image gallery when the user taps on the Image in the User Settings screen

## Links
* [MINI - 1694](https://jira.rakuten-it.com/jira/browse/MINI-1694) 
* [MINI - 1771](https://jira.rakuten-it.com/jira/browse/MINI-1771) 

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by corycaywood — APPROVED** [review]
👍 👍

File: Example/Controllers/UserSettingsTableViewController.swift [hunk 1]
```diff
@@ -30,6 +31,16 @@ class UserSettingsTableViewController: UITableViewController, UIImagePickerContr
         self.userProfileImage = profileImage
     }
 
+    func addTapGestureForImage() {
+        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(imageTapped(tapGestureRecognizer:)))
+        imageView.isUserInteractionEnabled = true
+        imageView.addGestureRecognizer(tapGestureRecognizer)
```
Comment (Climbatize) [inline Example/Controllers/UserSettingsTableViewController.swift:37]: why not using a button? Less code is better to me ^^

**Review by Climbatize — APPROVED** [review]
LGTM, we always have opposite visions on UI, but that's for the best :)

File: Example/Controllers/UserSettingsTableViewController.swift (hunk 1 above)
Comment (ts-leojoseph-r) [reply Example/Controllers/UserSettingsTableViewController.swift:37]: Lol. 👍 . Added in refactor To-Do's.

**Pull request merged**